code/go/internal/validator/spec.go

```diff
@@ -0,0 +1,53 @@
+package validator
+
+import (
+	"net/http"
+	"path"
+	"strconv"
+
+	"github.com/Masterminds/semver/v3"
+	_ "github.com/elastic/package-spec/code/go/internal/spec"
```

Comment:
nit: sort imports

Reply:
Fixed in 7741495a40da3ff61c5ab3c35bbcd61014e3f1a6